server/routerlicious/packages/services-core/src/lambdas.ts

```diff
@@ -86,13 +86,13 @@ export interface IPartitionLambda {
 /**
  * Factory for creating lambda related objects
  */
-export interface IPartitionLambdaFactory<T extends IPartitionConfig = IPartitionLambdaConfig>
+export interface IPartitionLambdaFactory<TConfig = undefined>
```

Comment:
There are other options here - this change defaults to no config, and requires explicitly expressing the config if needed (as opposed to before this change, defaulting to IPartitionLambdaConfig even if the factory didn't require any config).  LMK what you think.

Reply:
If the config used anywhere or is it always undefined. Maybe the argument should be removed

Reply:
Is the config used anywhere or is it always undefined? Maybe the config arg should be removed

Reply:
Yes - some of the lambdas use the IPartitionLambdaConfig, while others use no config.  But it's just these two configs currently.

Today the factories get that config by default, and they have to specify if they DON'T want it.  This change instead defaults to no config and those factories have to specify if they DO want it.  I do think there's probably better ways to structure this, but probably a bigger change.